config.json

```
@@ -180,7 +180,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+        {
+      "title": "Cross Platform",
+      "content": "Powershell runs on Windows, macOS, and Linux.",
+      "icon": "cross-platform"
+    },
+    {
+      "title": "Interop",
+      "content": "Leverage the .NET framework with ease.",
+      "icon": "interop"
+    },
+    {
+      "title": "Pipelines",
```

Pipelines sound like the implementation, not the actual feature

```suggestion
      "title": "Composable",
```